docs/modules/ROOT/pages/migration-guide-2.adoc

```diff
@@ -433,15 +433,15 @@ Passing parameters without spread will still return a defined type.
 new Cypher.Literal(`Hello "World"`);
 ----
 
-Would generate the following Cypher
+Would generate the following Cypher:
 
-Before: 
```

Comment:
Is this wrong?

Reply:
I changed it to be consistent with other parts of the file. But looking back it is probably better to change the other parts to match this one